docs/reference/transfers.md

```diff
@@ -18,6 +18,9 @@ creation.
 
 This is a unique identifier for the transaction.
 
+Values for account IDs *may* be reused as values for transfer IDs. You
```

Comment:
Maybe swap "account" and "transfer" here, since in the transfers doc the focus is shifted.

Reply:
Done